### PR TITLE
Epic - hide payment icons

### PR DESCRIPTION
**Pull request by KaliedaRik**
## What does this change?
Adds the ability to suppress the payment icons beneath the primary CTA by setting a new attribute

### Screenshots

**Epic (classic)**
Screenshot 2022-11-10 at 14 19 34

Screenshot 2022-11-10 at 14 19 48

**Epic with reminder**
Screenshot 2022-11-10 at 14 20 11

Screenshot 2022-11-10 at 14 20 24

**Epic with special header**
Screenshot 2022-11-10 at 14 20 45

Screenshot 2022-11-10 at 14 20 59

### CODE REVIEW

File: src/Epic/ContributionsEpicButtons.tsx [hunk 1]
```diff
@@ -123,8 +124,10 @@ export const ContributionsEpicButtons = ({
     remindMeConfirmationText,
     remindMeConfirmationHeaderText,
     trackClick,
+    hidePaymentIcons,
 }: ContributionsEpicButtonsProps): JSX.Element => {
     const [sectionState, setSectionState] = useState<SectionState>('DEFAULT');
+    const showPaymentIcons = hidePaymentIcons != null && hidePaymentIcons === 'true' ? false : true;
```
Comment (tjmw) [inline src/Epic/ContributionsEpicButtons.tsx:130]: I wonder, can this be simplified to:

`const showPaymentIcons = !(hidePaymentIcons === 'true');`

? I.e. the only time they should be hidden is if `hidePaymentIcons` equals the string `'true'` and in all other cases show them, we we flip the result of compaing to `'true'`.

**Review by tjmw — APPROVED** [review]
Looks great! One minor question/suggestion on the logic.

File: src/Epic/ContributionsEpicButtons.tsx (hunk 1 above)
Comment (tomrf1) [inline src/Epic/ContributionsEpicButtons.tsx:130]: is this equivalent to `const showPaymentIcons = hidePaymentIcons !== 'true'`?

Comment (tjmw) [reply src/Epic/ContributionsEpicButtons.tsx:130]: Ha I made (almost) the exact same suggestion above :-) This is better though!

Comment (KaliedaRik) [reply src/Epic/ContributionsEpicButtons.tsx]: Fixed with TomF suggested alternative

Comment (KaliedaRik) [reply src/Epic/ContributionsEpicButtons.tsx]: Fixed using this alternative code

**Pull request merged**